README.md

```diff
@@ -20,7 +20,7 @@ pre2release is an action to automatically finds repository's latest pre-release
     token: ${{ github.token }}
     
         
-    # Description:  You can add a prefix to semver e.g. v1.0.0 
+    # Description:  You can add a prefix to semver e.g. v2.0.0 
```

Comment:
Looks like this can just stay v1, example doesn't need to be updated

Reply:
Everything goes up a major version on the Node20 upgrade